Source/WebKit/WebProcess/Inspector/WebInspectorClient.cpp

```diff
@@ -69,53 +69,68 @@ WebInspectorClient::WebInspectorClient(WebPage* page)
 
 WebInspectorClient::~WebInspectorClient()
 {
+    RefPtr page = m_page.get();
+    if (!page)
+        return;
+
```

Comment:
To avoid behavior change, this should probably be moved to after `m_paintRectLayers.clear();`.